contracts/QuestFactory.sol

```diff
@@ -136,17 +136,6 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         revert QuestTypeInvalid();
     }
 
-    /// @dev grant the create quest role to an account
```

Comment:
we don't need this anymore?

Reply:
AccessControl has native `grantRole` and `revokeRole`!

https://docs.openzeppelin.com/contracts/4.x/api/access#AccessControl-grantRole-bytes32-address-

Reply:
➕ 1